api/SVGStyleElement.json

```
@@ -43,6 +43,40 @@
           "deprecated": false
         }
       },
+      "disabled": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGStyleElement/disabled",
+          "spec_url": "https://svgwg.org/svg2-draft/styling.html#__svg__SVGStyleElement__disabled",
+          "support": {
+            "chrome": {
+              "version_added": "102"
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "104"
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror"
+          },
+          "status": {
+            "experimental": true,
```

This now supported in multiple browsers.

```suggestion
            "experimental": false,
```